src/pages/SocialGraph.vue

```diff
@@ -153,27 +207,44 @@ export default {
       );
     },
 
-    async load() {
-      if (!typeMap.has(this.type)) return;
-      const type = typeMap.get(this.type);
+    async fetchPageData(page) {
```

Comment:
Maybe provide a default
```suggestion
    async fetchPageData(page = INITIAL_PAGE) {
```